src/test/util.js

```diff
@@ -30,18 +30,19 @@ import fs from 'fs-extra';
 
 import {exec} from 'child_process';
 import type {Test} from 'tape';
-
-import tmp from 'tmp';
+import uuid from 'uuid/v4';
 
 import {ThriftFileConverter} from '../main/convert';
 
+
 export const flowResultTest = (
   files: {[string]: string},
   testFn: (Function, FlowResult) => void,
   suffix: string = 'XXX',
   withsource: boolean = true
 ) => (t: Test) => {
-  const root = tmp.dirSync().name;
+  const root = path.resolve('test-output/', uuid());
```

Comment:
dep-free version: `crypto.randomBytes(20).toString('hex')` or `Math.random().toString(36).replace(/[^a-z]+/g, '')`

Reply:
Since it is a dev dep I prefer the dep version